plugins/samples/set_cookie/plugin.cc

```diff
@@ -54,15 +54,14 @@ class MyHttpContext : public Context {
     const auto cookies = getRequestHeader("Cookie")->toString();
     std::map<std::string, std::string> m;
     for (absl::string_view sp : absl::StrSplit(cookies, "; ")) {
-      m.insert(absl::StrSplit(sp, absl::MaxSplits('=', 1)));
+      const std::pair<std::string, std::string> cookie =
+          absl::StrSplit(sp, absl::MaxSplits('=', 1));
+      if (cookie.first == "my_cookie") {
```

Comment:
One more minor comment (sorry): since the cookie name is referenced in multiple places, can you define it as a constant?